google/cloud/aiplatform/models.py

```diff
@@ -3421,8 +3422,7 @@ def _deploy(
                 Optional. The full name of the Compute Engine network to which
                 this Endpoint will be peered. E.g. "projects/12345/global/networks/myVPC".
                 Private services access must already be configured for the network.
-                If left unspecified, the job is not peered with any network or
```

Comment:
This is actually not true on this method.

Reply:
Changed docstring to Endpoint instead of job. If left unspecified (kept as None), wouldn't the Endpoint not be peered with any network?

Reply:
The purpose of this comment was to capture that `_deploy` does not default to the `network` set in `init` but `deploy` does. 

ie: This line should be added back in for `deploy` and left out for `_deploy`:
```
If left unspecified, the job is not peered with any network or
                the network set in aiplatform.init will be used.
```

Preference to change to something like this because the statement above is slightly ambiguous:
```
If left unspecified, the network set in aiplatform.init will be used.
```

Reply:
Fixed. Left out the lines about `aiplatform.init(...)` in the helper functions and tried to be more specific with the docstrings for the public, main functions (`deploy` for Model and `create` for PrivateEndpoint)